package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Koa web app framework",
   "main": "lib/application.js",
   "scripts": {
-    "test": "make test"
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint benchmarks lib test"
```

Comment:
I don't think so, we also don't want to use a global eslint even if it exists (plugins).